misc/docusaurus/sidebars.js

```diff
@@ -0,0 +1,114 @@
+// @ts-check
+
+/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
+const sidebars = {
```

Comment:
Is or can this file be generated?

Reply:
Unfortunately, `sidebars.js` is a manual file we need to manage. It's practically the only moving part for Docusaurus that requires configuration and upkeep